Legion.FrontEnd/src/events/EventsCalendar/Components/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import moment from 'moment';
+import ChevronLeftIcon from 'mdi-react/ChevronLeftIcon';
+import ChevronRightIcon from 'mdi-react/ChevronRightIcon';
```

Comment:
I think you should be able to use object destructuring for these too:
```js
import { ChevronLeftIcon, ChevronRightIcon } from 'mdi-react';
```

Reply:
I'll try it.